File: 02-Patterns/pattern15/solution.py
"""

A B C D E
A B C D
A B C
A B
A

"""

alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"

for i in range(5):
    inc = 0
    for j in range(5, i, -1):
        print(alphabet[inc], end=" ")
        inc+=1
    print("")

File: 02-Patterns/pattern15/solution.js
/*

A B C D E
A B C D
A B C
A B
A

*/

const alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";

for (let i = 0; i < 5; i++) {
  let inc = 0;
  for (let j = 5; j > i; j--) {
    process.stdout.write(`${alphabet[inc]} `);
    inc += 1;
  }

  console.log("");
}
